x/cdp/simulation/operations.go

```diff
@@ -172,7 +174,8 @@ func SimulateMsgCdp(ak types.AccountKeeper, k keeper.Keeper, pfk types.Pricefeed
 
 			_, _, err := app.Deliver(tx)
 			if err != nil {
-				return simulation.NoOpMsg(types.ModuleName), nil, err
+				// to aid debugging, add the stack trace to the comment field of the returned opMsg
+				return simulation.NewOperationMsg(msg, false, fmt.Sprintf("%+v", err)), nil, err
```

Comment:
Mind adding this to all the modules? I've found the lack of logs when there's an error after `app.Deliver` to be a source of frustration.

Reply:
added